Makefile

```diff
@@ -60,11 +66,16 @@ ifeq ($(UI),1)
 	BUILD_TAGS += ui_server
 endif
 
-all: yaml image
+CLEAN_TARGETS :=
+
+CHAOS_MESH_BUILD_IN_DOCKER ?= 1
+ifeq (${IN_DOCKER},1)
+	CHAOS_MESH_BUILD_IN_DOCKER = 0
+endif
```

Comment:
why defining a new variable, can we use `IN_DOCKER` directly?

Reply:
`CHAOS_MESH_BUILD_IN_DOCKER ` means whether the user wants to build chaos-mesh binary in a docker container, which may fail for docker in VM because of the volume mount (like docker in Mac OS and windows). However, even with `CHAOS_MESH_BUILD_IN_DOCKER `, it may fail to build on Mac OS and windows (as the cgo is used).

But as the `--mount=cache` cannot be exported, and build an image depending on another image is not well supported by `buildx` (I cannot load a local image into the buildkit), I cannot find a way to keep the go build-cache while supporting building on mac os.

Reply:
Maybe I could fall back to the traditional way for Mac OS users 😋

Reply:
It has been removed.